process/css-styleguide/tag/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Tags / Pills</title>
+
+    <link rel="stylesheet" href="../heading/heading.css" />
+    <link rel="stylesheet" href="./tag.css" />
+    <link rel="stylesheet" href="../spacing.css" />
+    <style>
+      .tag-container--local {
+        padding: 1rem;
+      }
+    </style>
+  </head>
+
+  <body class="p-3">
+    <div class="tag-container--local">
+      <span class="tag tag-success typography body-s">active</span>
+      <br />
+      <br />
+      <span class="tag tag-error typography body-s">inactive</span>
+      <br />
+      <br />
+      <span class="tag tag-process typography body-s">
+        <span class="tag-loader"></span>
+        creating
+      </span>
+      <br />
+      <br />
+      <span class="tag tag-terminate typography body-s">deleted</span>
+    </div>

```

Comment:
You have here a mismatch between the text displayed and the class used: `success` vs `active`, `error` vs `inactive`, ...  It is not clear by reading it what would be displayed